src/App.vue

```diff
@@ -1,55 +1,159 @@
 <script lang="ts">
 import { defineComponent } from 'vue';
 import Main from './components/Main.vue'
+import MobileMenu from './components/MobileMenu.vue'
+import CartFlyout from './components/CartFlyout.vue';
 
 export default defineComponent({
   name: 'App',
+  components: {
+    Main,
+    MobileMenu,
+    CartFlyout
+  },
+  data(){
```

Comment:
Instead of using a data() function, you can define the type of your component's data with an interface